board/batocera/post-image-script.sh

```diff
@@ -407,6 +407,44 @@ case "${BATOCERA_TARGET}" in
         rm -f "${BATOCERA_BINARIES_DIR}/boot.vfat" || exit 1
         sync || exit 1
         ;;
+    TINKERBOARD)
+        # /boot
+        rm -rf "${BINARIES_DIR}/boot"            || exit 1
+        mkdir -p "${BINARIES_DIR}/boot/boot"     || exit 1
+	    mkdir -p "${BINARIES_DIR}/boot/extlinux" || exit 1
+        cp "${BINARIES_DIR}/zImage"                 "${BINARIES_DIR}/boot/boot/linux"                || exit 1
+        cp "${BINARIES_DIR}/initrd.gz"             "${BINARIES_DIR}/boot/boot/initrd.gz"            || exit 1
+        cp "${BINARIES_DIR}/rootfs.squashfs"       "${BINARIES_DIR}/boot/boot/batocera.update"      || exit 1
+        cp "${BINARIES_DIR}/rk3288-miniarm.dtb"  "${BINARIES_DIR}/boot/boot/rk3288-miniarm.dtb" || exit 1
+        cp "${BINARIES_DIR}/batocera-boot.conf"    "${BINARIES_DIR}/boot/batocera-boot.conf"        || exit 1
+	    cp "board/batocera/tinkerboard/boot/extlinux.conf" ${BINARIES_DIR}/boot/extlinux                   || exit 1
+        # boot.tar.xz
+        echo "creating boot.tar.xz"
+        (cd "${BINARIES_DIR}/boot" && tar -cJf "${BATOCERA_BINARIES_DIR}/boot.tar.xz" extlinux boot batocera-boot.conf) || exit 1
+
+	# blobs
+     MKIMAGE=$HOST_DIR/bin/mkimage
```

Comment:
I think this file has a mix of tabs and spaces for indentation. 
@nadenislamarre @jdorigao What should we do in those cases? I'm editing a file with the same problem.

Reply:
when there are only a view line to modify, i modify to fix.
you can do a commit to fix all a file if you want.
personnally, it is it not giving trouble, i let as it is

Reply:
If we decide to do commits to correct the fix those files, shoud we use tabs or spaces? :)